src/js/model.js

```diff
@@ -0,0 +1,13 @@
+export const data = {
+  todos: []
+}
+
+export const getLocalStorage = () => {
+  if (localStorage.getItem("data") !== null) {
+    data.todos = JSON.parse(localStorage.getItem("data")).todos
+  }
+}
+
+export const setLocalStorage = () => {
+  localStorage.setItem("data", JSON.stringify(data))
+}
```

Comment:
setItem의 key값을 data로만 사용해야할까?
이것도 함수동작이랑 매칭이 안됨
setLocalStorageData 정도로 해도 그나마 이해가 됐을듯

```suggestion
export const setLocalStorage = (key) => {
  localStorage.setItem(key, JSON.stringify(data))
}
```